include/Core/DB/TableManager.php

```diff
@@ -39,10 +39,7 @@ public function load() {
 	 */
 	public function on_activate( $network_wide ) {
 		if ( is_multisite() && $network_wide ) {
-			// Note: if there are more than 10,000 blogs or
-			// if `wp_is_large_network` filter is set, then this may fail.
-			// TODO: Take care of the deprecated function.
-			$sites = wp_get_sites();
```

Comment:
You should remove only the comment about the deprecated function. The other comment still holds good.